Utilities/build-script-helper.py

```diff
@@ -87,7 +87,7 @@ def delete_rpath(rpath, binary):
 
 
 def should_test_parallel():
-  if platform.system() == 'Linux':
+  if platform.system() == 'Linux' and 'ANDROID_DATA' not in os.environ:
     distro = platform.linux_distribution()
     if distro[0] != 'Ubuntu':
       # Workaround hang in Process.run() that hasn't been tracked down yet.
```

Comment:
I actually tracked down the bug this is working around after the comment was written and it is a bug in posix_spawn that only affected glibc-2.24.  I don't think we're specifically trying to support any distros that use the affected glibc right now, so I'd be okay with just removing this check.  If we do want to support this, the correct fix would probably be to patch corelibs-foundation's Process.run to not use posix_spawn on that glibc version (there is already a fallback implementation in foundation not using posix_spawn, but it's currently only enabled when posix_spawn is missing entirely).

Reply:
Removed it and added a comment with your info, in case somebody hits that later.